vhdx/src/vhdx.rs

```diff
@@ -0,0 +1,73 @@
+mod vhdx_header;
+mod vhdx_metadata;
+
+use crate::vhdx_header::{RegionTableEntry, VhdxHeader};
+
```

Comment:
Remove this blank line so that all imports will be sorted out properly when running `cargo fmt --all`.

Reply:
Done.